### PR TITLE
feat: get suggestions from address

### PR DESCRIPTION
**Pull request by pyphilia**

### CODE REVIEW

**Review by ReidyT — APPROVED** [review]
LGTM 👍

File: src/api/itemGeolocation.ts [hunk 1]
```diff
@@ -78,3 +80,19 @@ export const getAddressFromCoordinates = async (
       },
     )
     .then(({ data }) => data);
+
+export const getSuggestionsForAddress = async (
```
Comment (spaenleh) [inline src/api/itemGeolocation.ts:84]: Are these suggestions or coordinates ?

File: src/api/routes.ts [hunk 2]
```diff
@@ -500,6 +500,15 @@ export const buildGetAddressFromCoordinatesRoute = ({
 }: Pick<ItemGeolocation, 'lat' | 'lng'>) =>
   `https://nominatim.openstreetmap.org/reverse?format=jsonv2&lat=${lat}&lon=${lng}`;
 
+export const buildGetSuggestionsForAddressRoute = ({
+  address,
+}: {
+  address: string;
+}) => {
+  const d = new URLSearchParams({ address });
+  return `https://nominatim.openstreetmap.org/search?q=${d.get('address')}&format=json`;
```
Comment (spaenleh) [inline src/api/routes.ts:509]: Not sure why you use the `URLSearchParams` in that way ? Is it to `encodeURI` ? 
I think you could use this further to build the URL more cleanly:
```suggestion
  const query = new URLSearchParams({ q: address, format: 'json' });
  const url = new URL(`https://nominatim.openstreetmap.org/search`);
  url.search = query;
  return url.toString();
```

**Review by spaenleh — APPROVED** [review]
Looks Ok, see my suggestions :) ☂️

File: src/api/itemGeolocation.ts (hunk 1 above)
Comment (pyphilia) [reply src/api/itemGeolocation.ts:84]: Suggestions of addresses, there's address label, and also lat lng.

**Pull request merged**